### ISSUE TITLE
clean_after_parallel

### ISSUE THREAD

**Issue by crcrewso** [nrc-cnrc/EGSnrc#72]
When running `clean_after_parallel` I noticed the following behavior. It looks like the indices for the inputs are not lining up with `addphsp`'s expectations. I got the same behavior when either leaving the defaults as they are or manually entering in the parameters. 
```
Do you want to run addphsp on them? (y,[n]) y
 inputs required by addphsp:

 Enter P1=  ?.egsinp (default tb_10MV_steel_000cm)
tb_10MV_steel_000cm
 Enter P2=  splitnum (default 20)
20
 Enter P3=  _w? serial# to start (defaults to 1)
1
 Enter P4=  .egsphsp? scoring plane number (defaults to 1)
1

 Will sum from phsp file tb_10MV_steel_000cm_w1.egsphsp1
 to tb_10MV_steel_000cm_w1.egsphsp1
 And output result to 20.egsphsp1


 Output file 20.egsphsp1 exists.
 Add to it [add], overwrite it [ow], or quit [q--default]:
```

**Comment by crcrewso** [comment]
I think I solved it. @blakewalters, does this make sense???

I Changing one line in clean_after parallel from 

addphsp $p1 $p2 $p3 $p4

to

addphsp $p1 $p1 $p2 $p3 $p4 

(I'd send in a pull request but alas my dev box is not with me right now)

**Comment by blakewalters** [comment]
This makes sense.  The second input to addphsp is the base of the output file name, which, in this case, should be the same as the base of the input file name.

**Comment by crcrewso** [comment]
I just noticed that this version breaks if the user inputs anything other than default for the collection name. I've sent in a pull request using the tested solution addphsp $1 $p1 $p2 $p3 $p4 which relies on the command line input for the initial egsinp name.

**Comment by ftessier** [comment]
@blakewalters can you vouch for the fix proposed in PR #75?
